### PR TITLE
Way to add agent - change setProvider() method name to enable**()

### PR DESCRIPTION
**Pull request by Yuseon-Han**
solution no.2 of
https://tde.sktelecom.com/pms/browse/OPENSDK-82

Change way to add Agent

main purpose :
Some agent needs provider (or client) and others none.
Previously a way to add agent is different by these two type.
Now it has combined but It may cause uncomfortable to POC already serviced because It changed to call two functions.
So It changed again for letting POC to call just one function to add agent.

### CODE REVIEW

File: nugu-android-helper/src/main/java/com/skt/nugu/sdk/platform/android/NuguAndroidClient.kt [hunk 1]
```diff
@@ -411,32 +362,60 @@ class NuguAndroidClient private constructor(
...
+
+        /**
+         * @param client the provider for MessageAgent. If it is null, message agent not added.
+         */
+        fun enableMessage(client: MessageClient?) = apply { this.messageClient = client }
+
+        /**
+         * @param locationProvider the provider for LocationAgent. If it is null, location agent not added.
+         */
+        fun enableLocation(locationProvider: LocationProvider) = apply { this.locationProvider = locationProvider }
```
Comment (shshshl09) [inline nugu-android-helper/src/main/java/com/skt/nugu/sdk/platform/android/NuguAndroidClient.kt:412]: may be nullable?

File: app/src/main/java/com/skt/nugu/sampleapp/client/ClientManager.kt [hunk 2]
```diff
@@ -203,7 +203,7 @@ object ClientManager : AudioPlayerAgentInterface.Listener {
                 "skt_nugu_assets",
                 Context.MODE_PRIVATE
             ).absolutePath + File.separator + "skt_epd_model.raw")
-            .soundProvider(object : SoundProvider {
+            .enableSound(object : SoundProvider {
```
Comment (tommy5oo) [inline app/src/main/java/com/skt/nugu/sampleapp/client/ClientManager.kt:206]: It's a different issue, redundant semicolon..
```
                    return URI.create(
                        Uri.parse(ContentResolver.SCHEME_ANDROID_RESOURCE + "://" + context.packageName + "/" + R.raw.responsefail_800ms)
                            .toString()
                    );  <---
```

File: nugu-android-helper/src/main/java/com/skt/nugu/sdk/platform/android/NuguAndroidClient.kt (hunk 1 above)
Comment (Yuseon-Han) [reply nugu-android-helper/src/main/java/com/skt/nugu/sdk/platform/android/NuguAndroidClient.kt]: Changed to nullable in new commit

File: app/src/main/java/com/skt/nugu/sampleapp/client/ClientManager.kt (hunk 2 above)
Comment (Yuseon-Han) [reply app/src/main/java/com/skt/nugu/sampleapp/client/ClientManager.kt:206]: semicolon removed in new commit

**Review by tommy5oo — APPROVED** [review]

**Review by shshshl09 — APPROVED** [review]

**Pull request merged**